scripts/wawCommons.py

```diff
@@ -526,4 +526,85 @@ def replaceValue(sourceJson, target, replacementJson, matchKey = True):
             replacedValuesNumber += 1
     return targetJson, replacedValuesNumber
 
+def getFunctionResponseJson(CFNamespaceUrl, username, password, package, functionName, parameters, data):
+    functionCallUrl = CFNamespaceUrl + '/actions/' + package + '/' + functionName + parameters
+    logger.info("Calling function url '%s'", functionCallUrl)
+
+    functionResponse = requests.post(functionCallUrl, auth=(username, password),
+                                 headers={'Content-Type': 'application/json',
+                                          'accept': 'application/json'},
+                                 data=data)
+
+    if functionResponse.status_code == 200:
+
+        responseContentType = functionResponse.headers.get('content-type')
+        if responseContentType != 'application/json':
+            logger.error('Response content type is not json, content type: %s, response:\n%s', responseContentType, functionResponse.text)
+            return None
+
+        return functionResponse.json()
+
+    elif functionResponse.status_code == 202: # try once more
+        # 202 Accepted activation request (should not happen while sending 'blocking=true&result=true')
+        logger.warning("Did not receive response from function '%s' in package '%s', trying once more.", functionName, package)
```

Comment:
when user did not pass 'blocking=true&result=true' we should not print the warning... if we decide to pass params as map. it should be easy to check this

Reply:
Now this function always passes 'blocking=true&result=true'